manifest.yml

```diff
@@ -16,4 +16,8 @@ inputs:
     description: "Folder all media will be stored in. Defaults to Netlify site name"
   - name: uploadPreset
     required: false
-    description: "Defined set of asset upload defaults in Cloudinary"
\ No newline at end of file
+    description: "Defined set of asset upload defaults in Cloudinary"
+  - name: loadingStrategy
+    required: false
+    description: "The method in which Cloudinary loads images (Ex: lazy, eager)"
```

Comment:
minor change, can you just tweak this description "in which images are loaded" as this isn't relevant to Cloudinary itself